arbitrary-users-patch/Dockerfile

```diff
@@ -2,11 +2,8 @@ ARG FROM_IMAGE
 FROM ${FROM_IMAGE}
 USER 0
 # Set permissions on /etc/passwd and /home to allow arbitrary users to write
-RUN mkdir -p /home/user
-RUN chgrp -R 0 /home
-RUN chmod -R g=u /etc/passwd /home
 COPY [--chown=0:0] entrypoint.sh /
-RUN chmod +x /entrypoint.sh
+RUN mkdir -p /home/user && chgrp -R 0 /home && chmod -R g=u /etc/passwd /home && chmod +x /entrypoint.sh
```

Comment:
just a small note, it seems chmod is optional if file in git has +x permission already no ?

Reply:
This is true, but also it currently is `rw-r--r--`

Reply:
and it doesn't hurt to document in the dockerfile the permissions you want, rather than assuming the permissions in the filesystem will be used/respected/are correct.